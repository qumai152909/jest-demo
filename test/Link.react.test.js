// 使用React的test renderer和Jest的快照特性来和组件交互，获得渲染结果和生成快照文件

import React from 'react';
import CompCity from './Link.react';
import renderer from 'react-test-renderer';

test('CompCity test', () => {
  const component = renderer.create(
    <CompCity/>,
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
