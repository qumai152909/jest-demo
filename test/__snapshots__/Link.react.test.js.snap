// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CompCity test 1`] = `
<div
  className="comp-city"
>
  <h2>
    comp-city-title
  </h2>
  <div
    className="comp-body"
  >
    <h3>
      comp-body
    </h3>
  </div>
</div>
`;
